docs/restic.md

```diff
@@ -0,0 +1,91 @@
+# Restic Integration
+
+Ark has alpha support for backups/restores of pod volume data using [restic][1], an open-source tool for doing
```

Comment:
I'd remove
- alpha
- linux

Reply:
And also mention that names/commands/etc may change as we receive feedback and refine things